CHANGELOG.md

```diff
@@ -1,5 +1,30 @@
 # Confluent's Python client for Apache Kafka
 
+## v1.9.2
+
+v1.9.2 is a maintenance release with the following fixes and enhancements:
+
+ - KIP-140 Admin API ACL support:
```

Comment:
This suggest that ACL support was added, I suggest replacing "support" with "fix"